libbeat/reader/parser/parser.go

```diff
@@ -49,6 +49,8 @@ type CommonConfig struct {
 }
 
 type Config struct {
+	Suffix string
```

Comment:
maybe we want to name this field `stream`?

Reply:
I named it `Suffix` because from outside it is the string that is supposed to be added to the end of the state IDs. It does not matter if it contains a stream name or emojis describing the weather. The only thing that is relevant that it is a suffix.